app/services/common_platform_api/common_platform_connection.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module CommonPlatformApi
+  class CommonPlatformConnection < ApplicationService
```

Comment:
What do you think about renaming this class if the module names it effectively now?

Reply:
How about an `Api` and a `Connection` submodule under a `CommonPlatform` module?